### DISCUSSION TITLE
Database Images like Ergast

### DISCUSSION THREAD

**Discussion by Vucko95** [jolpica/jolpica-f1#97] [General]
Today I revisited project that uses data from Ergast after a year, I noticed that it has been deprecated. Firstly want to express my appreciation for your efforts in maintaining/reviving something simillar. 

There may be others like myself who use database snapshots of all data up to this point. Previously, Ergast provided large snapshots containing all relevant data after each race. However, I attempted to download the latest snapshot and found it to be empty.

The primary reason for my request is that my Formula 1 project will generate numerous requests, and I prefer to work with a local database to minimize the impact on the network of those providing this service.  

Additionally, I attempted to follow the documentation provided, but I encountered many challenges with getting GDAL to run on Windows, which has proven to be quite difficult. 

My question is :  Whether there are plans to implement a similar database snapshot feature in the future for this Project ?

![fsaaa](https://github.com/user-attachments/assets/8604a009-137b-4f9f-bba9-8a380f03a9f7)

**Comment by jolpica** [comment]
Hey,
We currently rely on the ergast dumps outselves, so we wont be providing dumps yet until at least next season. Also it is very likely that the dumps we provide will be different to Ergast's dumps (following our own DB schema).

Also see https://github.com/jolpica/jolpica-f1/discussions/91
